### PR TITLE
Feature small touches

### PR DESCRIPTION
**Pull request by RoelN**

### CODE REVIEW

File: src/fondue/Fondue.js [hunk 1]
```diff
@@ -48,6 +53,14 @@ export default class Fondue {
...
 	}
 
+	get charCount() {
+		return this.supportedCharacters.length;
+	}
+
+	get glyphCount() {
+		return this._font.opentype.tables.maxp.numGlyphs;
+	}
+
```
Comment (pascalw) [inline src/fondue/Fondue.js:63]: Can you add tests for these functions?

**Review by pascalw — COMMENTED** [review]
LGTM! (Except for the missing tests ;-))

File: src/fondue/Fondue.js (hunk 1 above)
Comment (RoelN) [reply src/fondue/Fondue.js:63]: I knew I forgot something that I forgot the previous 17 times also 🙈

**Comment by RoelN** [comment]
@pascalw Thanks for the review! ✊

**Pull request merged**